### PR TITLE
create vs coder page.

### LINKED ISSUE #2540: Create vs/coder landing page
See [Mockup](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/Website?node-id=6216%3A10404)

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #2540 

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2601"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2601"></a>

### CODE REVIEW

File: src/routes/vs/coder.svelte [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+/>
+
+<Header
+  title="Gitpod vs Coder"
+  text="Gitpod is the faster, more powerful productivity platform that integrates with your individual stack."
+  tight={true}
+  textClassNames="max-w-2xl text-large mx-auto"
+>
+  <LinkButton
+    href="/#try-now"
```
Comment (Siddhant-K-code) [inline src/routes/vs/coder.svelte:36]: Is it intentional? that, First Button of `Try Now` redirects to `/#try-now`(on the homepage) while the other button (below) redirects to `/workspaces`

Comment (nisarhassan12) [reply src/routes/vs/coder.svelte:36]: Great catch Siddhant. Thanks for bringing this up :+1:

**Comment by nisarhassan12** [comment]
@Siddhant-K-code I have not implemented the grouping of rows within this Pr yet because that will make the Pr huge so I have opened a follow-up issue for that https://github.com/gitpod-io/website/issues/2614

**Comment by Siddhant-K-code** [comment]
Yeah sure. I'll review this by EOD

File: src/lib/contents/coder.ts [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+  {
+    type: "text",
+    data: {
+      text: "Made for individuals",
+    },
+  },
+  {
+    type: "text",
+    data: {
+      text: "Pricing Saas",
```
Comment (Siddhant-K-code) [inline src/lib/contents/coder.ts:52]:
```suggestion
      text: "Pricing (SaaS)",
```

File: src/lib/contents/coder.ts [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+        {
+          term: "Made for teams",
+          availability: true,
+        },
+        {
+          term: "Made for individuals",
+          availability: true,
+        },
+        {
+          term: "Pricing Saas",
```
Comment (Siddhant-K-code) [inline src/lib/contents/coder.ts:196]:
```suggestion
          term: "Pricing (SaaS)",
```

File: src/lib/contents/coder.ts [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+        {
+          term: "Made for teams",
+          availability: true,
+        },
+        {
+          term: "Made for individuals",
+          availability: false,
+        },
+        {
+          term: "Pricing Saas",
```
Comment (Siddhant-K-code) [inline src/lib/contents/coder.ts:285]: I know, it won't be visible to the user. But, it should be consistent across the site.

```suggestion
          term: "Pricing (SaaS)",
```

File: src/lib/contents/coder.ts [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+  {
+    type: "text",
+    data: {
+      text: "Pricing Saas",
+    },
+  },
+  {
+    type: "text",
+    data: {
+      text: "Pricing Self-hosted",
```
Comment (Siddhant-K-code) [inline src/lib/contents/coder.ts:58]:
```suggestion
      text: "Pricing (Self-hosted)",
```

File: src/lib/contents/coder.ts [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
+          availability: true,
+        },
+        {
+          term: "Prebuilds",
+          availability: true,
+        },
+        {
+          term: "Snapshots",
+          availability: true,
+        },
```
Comment (Siddhant-K-code) [inline src/lib/contents/coder.ts:254]: Extra Pair of ✔️ ❌

image

**Review by Siddhant-K-code — CHANGES_REQUESTED** [review]

**Review by Siddhant-K-code — APPROVED** [review]
LGTM ✨ 
Thanks 🧡

**Pull request merged**